### DISCUSSION TITLE
failed to read artifact source file

### DISCUSSION THREAD

**Discussion by khandelwalmoksh** [Cyfrin/foundry-full-course-cu#2718] [Q&A] [answered]
Compilation Successful!, but a minute error
![Screenshot (23)](https://github.com/user-attachments/assets/04a66fab-6542-4cef-8b3d-2331011a4dee)
I hope it won't make any difference

**Comment by EngrPips** [comment, marked as answer]
Hello @khandelwalmoksh, run `forge clean` and things should work fine I think.
